CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/Services/ApiClient.cs

```diff
@@ -79,5 +79,41 @@ public async Task<HttpResponseMessage> PostAsync(string resource, string body)
                 throw;
             }
         }
+        public async Task<HttpResponseMessage> PutAsync(string resource, string body)
```

Comment:
We can directly deserialize the value returned from the API in the method and return a Generic parameter here. And accept a generic parameter for body and perform serialization and deserialization right here. so that the client would not bother about how our internal logic works.

`public async Task<TResult> PutAsync<TResult, TBody>(string resource, TBody body)`